docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-107.adoc

```diff
@@ -39,6 +39,8 @@ If your OpenAI API key is exposed, immediately regenerate a new key through your
 3. Find the API keys management page.
 4. If you believe your API key has been compromised or if you wish to invalidate an existing key for any reason, click on the option to regenerate a new API key.
```

Comment:
The first sentence stated that the key has been exposed. Therefore "If you believe your API key has been compromised or if you wish to invalidate an existing key for any reason," is redundant.
Do you generate a new key (and not regenerate)?

Reply:
Good callout. Updated to remove that and say "generate"